.github/workflows/build-test.yml

```diff
@@ -28,6 +28,8 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
+        with:
+          repository: chipsalliance/caliptra-dpe
```

Comment:
I think this checks out caliptra-dpe instead of caliptra-sw. Instead you should be able to change the "Initialize submodules" step below to just checkout dpe. Something like

`git submodule update --init runtime/dpe`